war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -837,6 +837,13 @@ function preventInputEe(event) {
   }
 }
 
+function escapeHTML(html) {
+  return html
+    .replace(/&/g, "&amp;")
+    .replace(/</g, "&lt;")
+    .replace(/>/g, "&gt;");
+}
+
```

Comment:
I had previously inlined this function in `hetero-list.js` and in a few places in this file (`makeButton` and `updatePos`). Those three usages should now be deduplicated to use this common function. If this is asking too much, I'll take care of this in a separate PR once this PR has landed.